dev/src/codewind/connection/InstallerWrapper.ts

```diff
@@ -64,9 +64,10 @@ namespace InstallerWrapper {
      */
     async function getInstallerStatus(): Promise<InstallerStatus> {
         const executablePath = await initialize();
+        Log.i("exectuablePath is " + executablePath);
 
         const status = await new Promise<InstallerStatus>((resolve, reject) => {
-            const child = child_process.execFile(executablePath, [ "status", JSON_OPTION ], {
+            const child = child_process.execFile(executablePath, [ "--insecure", "status", JSON_OPTION ], {
```

Comment:
curious what does this option do?

Reply:
It's used when the installer makes a REST connection to pfe, it's the same as `--insecure` on curl - it lets us connect when there's an https connection with a self signed cert. (I thought this change would have been in master too, so I don't mind removing it.)